internal/pkg/override/cdk.go

```diff
@@ -140,11 +139,20 @@ func (cdk *CDK) cleanUp(body []byte) ([]byte, error) {
 }
 
 // ScaffoldWithCDK bootstraps a CDK application under dir/ to override the seed CloudFormation resources.
+// If the directory is not empty, then returns an error.
 func ScaffoldWithCDK(fs afero.Fs, dir string, seeds []template.CFNResource) error {
+	// Ignore errors on purpose. IsEmpty returns false if the directory does not exist, hence we should only check
+	// if the dir is empty if it also exists.
+	exists, _ := afero.Exists(fs, dir)
+	isEmpty, _ := afero.IsEmpty(fs, dir)
```

Comment:
Is ignoring errors here is safe? Are we sure that there wouldn't be a case where some genuine error or panic won't occur or won't cause us to fail?

Reply:
I've replied here: https://github.com/aws/copilot-cli/pull/4572/files#r1127021722

Does that help?